src/frontends/tensorflow_common/src/op/reciprocal.cpp

```diff
@@ -14,11 +14,13 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_reciprocal_op(const NodeContext& node) {
+    // computes element-wise 1/x, where x - input
+    default_op_checks(node, 1, {"Reciprocal"});
     auto x = node.get_input(0);
-    auto ng_exponent = make_shared<Constant>(x.get_element_type(), Shape{}, -1);
-    auto res = make_shared<Power>(x, ng_exponent);
-    set_node_name(node.get_name(), res);
-    return res->outputs();
+    auto minus_one_const = make_shared<Constant>(x.get_element_type(), Shape{}, -1);
```

Comment:
Shape{1} or Shape{} is correct?

Reply:
a scalar (i.e. `Shape{}`) is the correct value because the input can be a scalar.